### ISSUE TITLE
I am not able to login

### ISSUE THREAD

**Issue by davidnalbandian143** [ahmetbcakici/DynamicPersonalWebsite#1]
Not able to login with "admin" and "123" credentials

**Comment by ahmetbcakici** [comment]
Are you trying to login to the project via my website or did you install it on your computer?

**Comment by davidnalbandian143** [comment]
I tried in both ways but unable to logged in. It is showing **"Incorrect Username or Password!"** . I tried many times but no use

**Comment by davidnalbandian143** [comment]
Can you please help me out ?

**Comment by ahmetbcakici** [comment]
There is problem with my login system & hashing password. I will check it and solve problems soon.

**Comment by davidnalbandian143** [comment]
How do i access admin without login temporarily  @ahmetbcakici

**Comment by davidnalbandian143** [comment]
Please help me out

**Comment by ahmetbcakici** [comment]
I'm working on another project. I will take a look when i'm available as i said. Is there an urgent situation?

**Comment by davidnalbandian143** [comment]
Yes bro, Urgent , I think it is small issue but i can't figure it out some how.  Hope you understand

**Comment by PrathamBumb** [comment]
I am able to restore database files and able to login

**Comment by RBMdeveloper** [comment]
can you explain the steps @PrathamBumb

**Comment by NIGHTX-99** [comment]
not able to login using admin & 123
